### ISSUE TITLE
Snapped Items - Build List of Snapped Lines to Points

### ISSUE THREAD

**Issue by solutegrate** [codeofsumit/leaflet.pm#494]
Let's say I have (2) Lines snapped to (1) point.

Each time I snap a line to a point, can you push the Leaflet_ID of the line to the point Layer on the Feature Properties?

Example:

In the background, when I create a feature (let's say a point), I auto push in the feature.properties with the Leaflet_ID, Feature Type, etc. etc. At this same time, I'd like to add an array called "Snapped Lines" and each time a line is snapped to this point, add the Leaflet_ID from the line into this array.

Basically, building a relationship array on the point. That way I can reference all lines snapped to a point.

Then vice versa on the lines, I would like to see all points associated with the selected line.

Not sure if this already exists, but this would complete my project I'm working on if you can get that to work!

Thanks again for the awesome work

**Comment by codeofsumit** [comment]
That’s a fantastic suggestion and would basically be the groundwork for #493 👌

**Comment by solutegrate** [comment]
Sweet! Let me know if you need any other suggestions/help. I work with massive geo databases and map applications for (can't say their name, but "Can you hear me now?")

**Comment by codeofsumit** [comment]
@solutegrate I'm interested to learn more about your use cases. Would you contact [me via email](mailto:sk@outlook.com)?

**Comment by solutegrate** [comment]
I sent you an email. Also I have another question (Hopefully easy!)

How do I add the footage to the line being drawn and edited? As I'm drawing/editing, I would like to see the footage/miles/etc on top of the line or in the popup.

**Comment by solutegrate** [comment]
Nevermind. I wasn't thinking when I asked that question. here it is:

`map.on('mousemove', function (e) {
		    	var from = turf.point([vertexLatLng.lat, vertexLatLng.lng]);
				var to = turf.point([e.latlng.lat, e.latlng.lng]);
				var options = {units: 'miles'};
				var distance = turf.distance(from, to);

		    	console.log(distance * 3280.8);
			});`

**Comment by eharason** [comment]
> That’s a fantastic suggestion and would basically be the groundwork for #493 👌

ahh that's exactly what I'm working on right now. How did you implement what's in your screen capture? 😄

**Comment by codeofsumit** [comment]
@eharason It will be a new mode where you can basically pin shared vertices. Then all layers with the same vertex will stay pinned to the edited one.
You can turn this on and off (just like snapping) whenever needed.

**Comment by eharason** [comment]
right on!

**Comment by codeofsumit** [comment]
This is done now: #493 
https://geoman.io/blog/pinning-now-in-leaflet-geoman-pro
